src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/data/GraphQLProperty.java

```diff
@@ -0,0 +1,24 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data;
+
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLFieldType;
+
+/**
+ * Used to represent information about a field in a GraphQL object
+ */
+public interface GraphQLProperty {
```

Comment:
If this is about fields, then it shouldn't be named "property"! That's highly confusing.